src/schema/schema.rs

```diff
@@ -104,10 +105,199 @@ impl Schema {
           Some(error) => error
         }
       },
-      (&SchemaType::Enum(_), &Query::Keys(_)) => Err(Error::validation("Cannot deeply query an enum.", NO_PRIMITIVE_HINT)),
+      (&SchemaType::Enum{..}, &Query::Keys(_)) => Err(Error::validation("Cannot deeply query an enum.", NO_PRIMITIVE_HINT)),
+      (&SchemaType::None, _) |
       (_, &Query::All) => Ok(())
     }
   }
+
+  /// Create a schema which does not run any validations.
+  pub fn none() -> Self {
+    Schema {
+      type_: SchemaType::None
+    }
+  }
+
+  /// Create a schema which ensures the value is null and only null.
+  pub fn null() -> Self {
+    Schema {
+      type_: SchemaType::Null
+    }
+  }
+
+  /// Create a schema which validates a boolean primitive type.
+  pub fn boolean() -> Self {
+    Schema {
+      type_: SchemaType::Boolean
+    }
+  }
+
+  /// Create a schema which validates a number.
+  pub fn number() -> Self {
+    Schema {
+      type_: SchemaType::Number {
+        multiple_of: None,
+        minimum: None,
+        exclusive_minimum: false,
+        maximum: None,
+        exclusive_maximum: false
+      }
+    }
+  }
+
+  /// Create a schema which validates an integer.
+  pub fn integer() -> Self {
+    Schema {
+      type_: SchemaType::Number {
+        multiple_of: Some(1f32),
+        minimum: None,
+        exclusive_minimum: false,
+        maximum: None,
+        exclusive_maximum: false
+      }
+    }
+  }
+
+  pub fn set_multiple_of(&mut self, multiple_of: f32) -> bool {
```

Comment:
Currently you have to define all the methods to get information out of `SchemaType::String` for all the other possible values of `SchemaType`. This is a major break of encapsulation.